### PR TITLE
JANITORIAL: Typo squashing phase 1

### PR DESCRIPTION
**Pull request by raziel-**
Second try...

- Most of the typos are comment based
- Four are variables i think (but i believe i caught all of it's instances)
- Three are warning/error outputs
- Two are in-game talk lines
- Also added two FIXME comments in create_dat files
(those will change the created .dat files, i'll leave them to the devs)

If i should (still) continue, i'll tackle the engines next (many, many typos there)
If not, i'll go back to my dark, warm, happy place...with a blanket...and lots of chocolade...for real

Thank you

### CODE REVIEW

**Comment by sev-** [comment]
Looks much better now, thank you!

Are you able to remove commits? Because I'd like you to skip any changes in `backends/imgui` because that is the upstream code. If you like, you may create a PR in their repository: https://github.com/ocornut/imgui

File: devtools/create_mortdat/gametext.h [hunk 1]
```diff
@@ -406,6 +406,7 @@ const char *gameDataEn[] =  {
 	"Apart from the letter for you I posted, nothing very important!$",
 	"I was very happy she gave me her bound bible as a present$",
 	"It happened fast and she didn't have time to make any particular will$",
+	// FIXME: Spelling incorrect. suprised should be surprised. Fixing changes game data and thus may cause issues
```
Comment (sev-) [inline devtools/create_mortdat/gametext.h:409]: This is fine. We may regenerate the data.

**Comment by sev-** [comment]
Besides the imgui/ changes which I'd like to avoid, so we could easily upgrade the library later, the rest looks good to me. Thank you for looking into it!

File: devtools/create_mortdat/gametext.h (hunk 1 above)
Comment (raziel-) [reply devtools/create_mortdat/gametext.h:409]: There is another FIXME a fewl7nes above this...not from me though

**Comment by raziel-** [comment]
i don't think i can remove commits...other than revert the changes and cause two or more extra commit messages...if that is fine?

**Comment by raziel-** [comment]
imgui fixes reverted

File: backends/text-to-speech/windows/windows-text-to-speech.cpp [hunk 2]
```diff
@@ -365,7 +365,7 @@ void WindowsTextToSpeechManager::createVoice(void *cpVoiceToken) {
 	}
 
 	if (desc == "Sample TTS Voice") {
-		// This is really bad voice, it is basicaly unusable
+		// This is really a bad voice, it is basically unusable
```
Comment (meekee7) [inline backends/text-to-speech/windows/windows-text-to-speech.cpp:368]: As we are already making petty changes here, I'd suggest rewording this to "a really bad voice". That sounds more natural to me.

Comment (raziel-) [reply backends/text-to-speech/windows/windows-text-to-speech.cpp]: Everything to please the crowd

**Comment by sev-** [comment]
Thank you!
